From 58f79fed0ef8b90db18d963da7de1964dbe6eecc Mon Sep 17 00:00:00 2001
From: Kent Overstreet <kent.overstreet@linux.dev>
Date: Mon, 11 Dec 2023 15:08:05 -0500
Subject: [PATCH 63/74] x86/signal: kill dependency on time.h

this is unecessary, and was pulling in printk.h from uapi headers

Signed-off-by: Kent Overstreet <kent.overstreet@linux.dev>
---
 arch/x86/include/uapi/asm/signal.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/include/uapi/asm/signal.h b/arch/x86/include/uapi/asm/signal.h
index 777c3a0f4..f77734645 100644
--- a/arch/x86/include/uapi/asm/signal.h
+++ b/arch/x86/include/uapi/asm/signal.h
@@ -4,7 +4,6 @@
 
 #ifndef __ASSEMBLY__
 #include <linux/types.h>
-#include <linux/time.h>
 #include <linux/compiler.h>
 
 /* Avoid too many header ordering problems.  */
-- 
2.43.0.rc2

